### PR TITLE
Users

### PR DESCRIPTION
**Pull request by n0rdie**
## Description
User CRUD

## What type of PR is this? (check all applicable)
- [x] 💡💫 Feature
- [ ] 🐞🐛 Fix
- [x] 🪸🎭 Refactor
- [ ] 💅🎨 Style
- [ ] 📄💾 Documentation

## Reviewers
Dylan, Isaac, Quin

## Changes
List the changes introduced by this pull request.
- User Create
- User Delete
- User Show
- More testing for Creator Create

## How Has This Been Tested?
Describe testing implemented and what all it covers.
- POST Creator Create [SAD]-no name given
- POST Creator Create [SAD]-blank name given
- POST User Create
- POST User Create [SAD]-name already in use
- POST User Create [SAD]-no name
- POST User Create [SAD]-blank name
- DELETE User Delete
- DELETE User Delete - [SAD]-bad id
- User Show
- User Show [SAD]-bogus id

## Related Issues
Indicate any related issues that this pull request addresses or resolves.

## Checklist
- [x] I have read the contribution guidelines.
- [x] I have tested the changes locally.
- [x] The code follows the project's coding standards.
- [x] Documentation has been updated (if applicable).
- [x] I have added/updated unit tests (if applicable).
- [x]  I have squashed/organized my commits.

## Additional Notes
Add any additional information that might be relevant for reviewers.

### Thanks, GO TEAM! 👏

### CODE REVIEW

File: app/controllers/api/v1/users_controller.rb
Comment (tmitchellisaac) [inline app/controllers/api/v1/users_controller.rb:1]: Nice

File: spec/models/user_spec.rb
Comment (tmitchellisaac) [inline spec/models/user_spec.rb:1]: Yeah was gonna ask if we were going to keep email as an attribute?

File: log/test.log
Comment (tmitchellisaac) [inline log/test.log:1]: Logggg

File: config/routes.rb
Comment (tmitchellisaac) [inline config/routes.rb:1]: Nice change caught this on my branch too

File: app/controllers/api/v1/creators_controller.rb
Comment (tmitchellisaac) [inline app/controllers/api/v1/creators_controller.rb:1]: Nice sad paths 🙏

**Review by tmitchellisaac — APPROVED** [review]

File: spec/models/user_spec.rb
Comment (n0rdie) [reply spec/models/user_spec.rb:1]: I think we should keep it but have in not required

**Pull request merged**